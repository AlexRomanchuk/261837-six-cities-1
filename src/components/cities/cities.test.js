import React from "react";
import renderer from "react-test-renderer";
import Cities from "../cities/cities.jsx";

const mocCities = [`Amsterdam`, `Dusseldorf`, `Moscow`];
const currentCity = `Amsterdam`;

it(`correct renders main page`, () => {
  const tree = renderer
    .create(<Cities
      data={mocCities}
      currentCity={currentCity}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
