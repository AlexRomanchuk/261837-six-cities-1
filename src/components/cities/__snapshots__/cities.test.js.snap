// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correct renders main page 1`] = `
<section
  className="locations container"
>
  <ul
    className="locations__list tabs__list"
  >
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item tabs__item--active"
        href="#"
      >
        <span>
          Amsterdam
        </span>
      </a>
    </li>
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item"
        href="#"
      >
        <span>
          Dusseldorf
        </span>
      </a>
    </li>
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item"
        href="#"
      >
        <span>
          Moscow
        </span>
      </a>
    </li>
  </ul>
</section>
`;
